hack/.golint_failures

```diff
@@ -3,6 +3,7 @@ cmd/kube-apiserver/app
 cmd/kube-controller-manager/app
 cmd/kube-proxy/app
 cmd/kubeadm/app/apis/kubeadm/v1beta1
+cmd/kubeadm/app/apis/kubeadm/v1beta2
```

Comment:
This moves us in the wrong direction.

Please address the lint errors in your code and remove the exception on this package.

Reply:
my research on the problem here is the code generated by the k8s code generators do not pass the current state of go linters. they use underscores in generated function names such `SetDefaults_MyFunction`, which does not pass.
https://github.com/kubernetes/kubernetes/pull/76710/files#diff-9a2f5c3d2d4df5c049e0951f45d50133R69

this means that API namespaces need to be skipped.

Reply:
@neolit123 is right here. I am not aware of any way of making the defaulter gen call funcs, that don't have underscores in their name and these obviously break the linter.

Reply:
Is there an issue to fix this in the code generators?

Reply:
this PR seems to fix comment related lint issues, but not the `_`:
https://github.com/kubernetes/kubernetes/pull/59674

this ticket has a good enough title to be re-used:
https://github.com/kubernetes/code-generator/issues/30

i will re-open it and explain the `_` case.

Reply:
Thanks!